workspaces/icecap-host/rust-toolchain.toml

```diff
@@ -0,0 +1,4 @@
+[toolchain]
```

Comment:
TIL!  Is this a new feature?

Reply:
Yes: https://blog.rust-lang.org/2021/04/27/Rustup-1.24.0.html

This is not new for Veracruz: https://github.com/veracruz-project/veracruz/blob/fada738c9a73c93aa63c773bba57a0b7d6fc79d4/icecap/rust-toolchain.toml